fetchmail_notify_error_to_sender/models/mail_thread.py

```diff
@@ -21,7 +21,7 @@ def message_route(
                 thread_id=thread_id,
                 custom_values=custom_values,
             )
-        except ValueError as ve:
+        except Exception as ve:
```

Comment:
why? https://github.com/odoo/odoo/blob/16.0/addons/mail/models/mail_thread.py#L1118

Reply:
Before I had tested it, It wasn't going inside the ValueError. To Catch all the exceptions I added the Exception. I've changed it to ValueError. Please review it. 
Thank you.